app/views/check_file.scala.html

```diff
@@ -37,15 +37,15 @@ <h1 class="heading-medium error-summary-heading" id="error-summary-heading">
 		</ul>
 	</div>
```

Comment:
same comments as last error summary, we should also move it to above the page heading for consistency and for it to follow the current desig pattern

Reply:
check_file and check_csv_file both contain errors that are never normally visible to a user, as the functionality directs the user to a separate error page should they encounter any issues. I left those that are never visible untouched after advice to leave the functionality change to a separate ticket - figured any change that I made would likely need to be overridden then either way

Reply:
I have no issues with that